modules/touch/src/touchinteraction.cpp

```diff
@@ -425,6 +430,17 @@ void TouchInteraction::updateStateFromInput(const std::vector<TuioCursor>& list,
     }
 }
 
+bool TouchInteraction::webContent(const std::vector<TuioCursor>& list) {
+    glm::ivec2 res = global::windowDelegate.currentWindowSize();
+    glm::dvec2 pos = glm::vec2(
+        list.at(0).getScreenX(res.x),
+        list.at(0).getScreenY(res.y)
+    );
+
+    WebBrowserModule& module = *(global::moduleEngine.module<WebBrowserModule>());
```

Comment:
Wrapping this in #ifdef OPENSPACE_MODULE_WEBBROWSER_ENABLED would make this compile without the webbrowser

Reply:
Done